handbook/engineering/web/index.md

```diff
@@ -39,17 +39,26 @@ We will tackle these medium-term goals in order, though expect to have some work
 
 ### Short term
 
-The web team's short-term goals are documented in [the tracking issue for the current milestone](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+label%3Atracking+label%3Ateam%2Fweb+is%3Aopen).
+See [iterations](#iterations).
 
 ## Tech stack
 
 TypeScript, React, RxJS, GraphQL, Go.
 
 ## Processes
 
-### Planning
+### Iterations
 
-We do [planning](../tracking_issues.md#planning-a-milestone-with-a-tracking-issue) with [tracking issues](../tracking_issues.md).
+We plan our work in **2-week iterations**.
+We work on a single project in those 2 weeks as a team.
+
+Our goals for the current and past iterations can be found in our [iteration goals living Google Doc](https://docs.google.com/document/d/1n9WKjieKmd2YYkNrEsOfdmxRYUrbowLWjq05phLoQ6s/edit).
+
+The individual tasks and progress of the current iteration can be found as GitHub issues in our [GitHub project board](https://github.com/orgs/sourcegraph/projects/45?fullscreen=true), with estimate labels and assignees.
+At the beginning of an iteration, we fill the "Planned" column with all issues we want to accomplish by the end of the iteration and make sure they have estimates and assignees.
+Each issue should be as small as possible.
+Over the course of the iteration, we strive to close issues with small, iterative pull requests that we review and merge timely (generally a day).
+By the end of an iteration, all issues should be in the "Done" column and the board gets emptied.
```

Comment:
Should these be bullets?

```suggestion
- The individual tasks and progress of the current iteration can be found as GitHub issues in our [GitHub project board](https://github.com/orgs/sourcegraph/projects/45?fullscreen=true), with estimate labels and assignees.
- At the beginning of an iteration, we fill the "Planned" column with all issues we want to accomplish by the end of the iteration and make sure they have estimates and assignees.
- Each issue should be as small as possible.
- Over the course of the iteration, we strive to close issues with small, iterative pull requests that we review and merge timely (generally a day).
- By the end of an iteration, all issues should be in the "Done" column and the board gets emptied.
```

Reply:
Do think they're better as bullets compared to a prose paragraph?

Reply:
I don't feel strongly - I thought the intention was for them to be bullets since each sentence started on it's own line. Do what is most readable to you!

Reply:
Oh, no I just write one line per sentence for better git diffs/less merge conflicts. Single line breaks are ignored in markdown so it's one paragraph when rendered. I'd change it to bullets though if you think that's more readable :)

Reply:
Leaving it as prose is fine with me!